src/cpp/include/openvino/genai/whisper_generation_config.hpp

```diff
@@ -56,6 +53,18 @@ class OPENVINO_GENAI_EXPORTS WhisperGenerationConfig {
 
     bool is_multilingual = true;
```

Comment:
is it something that user wants to change?

For me, it seems that some read-only parameters are mixed with read-write parameters in whisper generation config, how do you think? I wonder whether I want to change lang_to_id mapping ?

Reply:
HuggingFace allows to set `is_multilingual` as a generate flag. But doesn't accepts `lang_to_id`, so `lang_to_id` could only be changed in generation_config.json.
At the moment we have `WhisperGenerationConfig` default constructor, so we have to provide API to set all the fields.
We could remove it and rely solely on initialization from `generation_config.json`. Then we could set `is_multilingual` and `lang_to_id` as read-only parameters.
The reason I see to keep default constructor is to skip `generation_config.json` reading from disk. Could save tens of microseconds. Seems as an advanced scenario, yeah.

Reply:
I am just thinking about  simplifying and expose only meaningful set of options in generation config, which can be changed by users.
Maybe other options like lang_to_id can go to WhisperPipeline::Config which is not changed by user? by used by pipeline

Reply:
I will look into that. It seems we can expose things like `task`, `language`, `return_timestamps` + sampling parameters. And hide all of those `decoder_start_id`, etc. Don't you mind I'll address it in further PRs? Todo added.

Reply:
> I will look into that. It seems we can expose things like `task`, `language`, `return_timestamps` + future sampling parameters. And hide all of those `decoder_start_id`, etc. Don't you mind I'll address it in further PRs? Todo added.

sure